install-elasticsearch.rst

```diff
@@ -148,30 +111,56 @@ Ubuntu 16.04 & 18.04:
  systemctl restart elasticsearch
  systemctl enable elasticsearch
 
+Adjust default settings of Elasticsearch
+++++++++++++++++++++++++++++++++++++++++
+
+.. Note:: The we found the below settings to work good with Zammad. Please note that this is only suggestion that can affect your local environment.
+
+To ensure an optimal performance of Zammad together with elasticsearch, you might want to increase the maximum possible 
+content length for http requests by adding the following to your ``/etc/elasticsearch/elasticsearch.yml``:
+
+::
+  
```

Comment:
You have trailing white space here and many other places. It doesn't really affect the end result (because the ReStructuredText engine will ignore it); it's just invisible clutter.

Git will show you trailing whitespace when you `git diff` or `git show`:

![Screenshot_20200110_175946](https://user-images.githubusercontent.com/12194123/72144280-452e4e00-33d3-11ea-8aa5-d8cfecc380cf.png)

Sometimes this happens because your text editor or IDE automatically indents a line when you hit Enter, and then you don't put any text on it but the indentation doesn't get automatically removed when you move elsewhere. Some text editors can be configured to highlight trailing white space, and you can usually use a find-and-replace-with-regex (`/\s+$/`) to get rid of all of them across whole files/projects.

In any case, I just wanted to let you know. I'm kind of anal-retentive about stuff like this, but in the future, I will fix it without saying anything.